### DISCUSSION TITLE
Fetch Latest Tweets

### DISCUSSION THREAD

**Discussion by adelghaenian** [WeAreMahsaAmini/Core#53] [Ideas]
On the homepage, we can embed tweets containing the #مهسا_امینی and present them on the site.

**Comment by ramyareye** [comment]
I think we should show trending information, tweets are accessible through twitter platform!

**Comment by ghost** [comment]
#no_to_sedition
#نه_به_آشوب
#نه_به_اغتشاش
#صلح
#مرگ_بر_آشوبگر
